modules/engines/engine-hdb/src/test/java/com/sap/xsk/hdb/ds/test/itest/utils/TestConstants.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.hdb.ds.test.itest.utils;
+
+import org.eclipse.dirigible.commons.config.Configuration;
+
+public class TestConstants {
```

Comment:
It's good idea to have such class, but it should contain only constants that are reused inside multiple test classes. Let's put the test specific constants inside the test class. In the end maybe only generic constants like `HANA_DRIVER`, `HANA_URL`, `HANA_USERNAME`, `HANA_PASSWORD`, etc. should be left here.

Reply:
Test specific constants are now removed except for the hana ones.